tests/testing_projects/make_based/Makefile

```diff
@@ -2,9 +2,23 @@
 CC = gcc
 CFLAGS = -O2 -std=c99
 
-.PHONY: all clean
+.PHONY: default with-assembly with-linking clean
 
-all: file.o
+default: file.o
+
+# Assembly files should not be compiled to `.ll`
+with-assembly: default mod.o sub.o
```

Comment:
Aren't we missing a rule for the `mod.o` target? Ot is it implicit?

Reply:
It is implicit. For `sub.o` (`sub.s`) I wrote it explicitly because implicitly would be used `as` (assembler) which would mean that `cc_wrapper`would not be called.  For `mod.o` (`mod.S`) is implicitly used `$(CC)`.

Reply:
Thanks for the explanation. Let's put it to a comment as I'm sure that I'll have the same question when I look at the code in future.